projects/observability/src/pages/apis/api-detail/api-detail.service.ts

```diff
@@ -18,8 +19,9 @@ export class ApiDetailService extends EntityDetailService<ApiEntity> {
   }
 }
 
-export interface ApiEntity extends Entity {
+export interface ApiEntity extends Entity, Breadcrumb {
```

Comment:
having ApiEntity extend breadcrumb doesn't seem intuitive. Does this service actually return all the breadcrumb info? It doesn't look like it.

Reply:
Yeah, we shouldn't really be using APIEntity in breadcrumb resolver but I am trying to limit the refactor in the PR.